### ISSUE TITLE
Error: AH95906 added after current Hub snapshot date.

### ISSUE THREAD

**Issue by lssb** [grisslab/scAnnotatR#1]
I am following the basic guide but getting a Error:

```
> default_models <- load_models("default")
  |================================================================================================| 100%

snapshotDate(): 2021-05-18
Error: AH95906 added after current Hub snapshot date.
  added: 2021-09-10
  snapshote date: 2021-05-18
```

**Comment by lssb** [comment]
How to deal with this?

**Comment by nttvy** [comment]
Hi @lssb

May we ask you which Bioconductor version you are using? It might be the problem with the Bioc version.

Best regards,
Vy

**Comment by lssb** [comment]
Hi @nttvy , Thanks for your response.

I checked the version of my Bioc, and found it was out of date. 

Now I am updating Bioc and I'll try again later.
```
> library(BiocManager)
Bioconductor version 3.13 (BiocManager 1.30.16), R 4.1.1 (2021-08-10)
Bioconductor version '3.13' is out-of-date; the current release version '3.14'
  is available with R version '4.1'; see https://bioconductor.org/install
```

**Comment by nttvy** [comment]
I guess this problem has been solved, so I close the issue. Please reopen it if needed.
